client/src/js/services/util.js

```diff
@@ -108,19 +108,17 @@ function UtilService($filter) {
 
   // Define the maxLength By Value
   service.length250 = 250;
-  service.length200 = 200;  
+  service.length200 = 200;
   service.length150 = 150;
   service.length100 = 100;
   service.length70 = 70;
   service.length50 = 50;
-  service.length50 = 45;  
+  service.length50 = 45;
   service.length40 = 40;
   service.length30 = 30;
-  service.length20 = 20;  
-  service.length16 = 16;  
+  service.length20 = 20;
```

Comment:
Can these lengths be specified by the client passing a parameter into the service? I understand this is just a minor whitespace refactor so is not relevent to this pull request. 
@IMA-WorldHealth/local-contributors